### DISCUSSION TITLE
How to show an image in HtmlView using an <img> tag

### DISCUSSION THREAD

**Discussion by mark-summerfield** [fltk-rs/fltk-rs#1349] [Q&A] [answered]
I want to show an image in an HtmlView using an <img> tag. But I want the image to be from inside the executable (e.g., an SVG image loaded at compile time with `include_str!("myimage.svg")`. Is this possible?

**Comment by MoAlyousef** [comment, marked as answer]
I don't think so. Checking the implementation of Fl_Help_View, it seems it uses the src attribute to look for an image in the filesystem.

**Reply by mark-summerfield** [reply]
Ok, thanks. Seems a strange omission from FLTK given that it supports images from strings or bytes.
